install_config/aggregate_logging.adoc

```diff
@@ -102,309 +90,419 @@ deployed. To control component placement, specify node selectors per component t
+for deploying Curator instances.
+|===
 
-Replace `<tag>` with `{latest-tag}` for the latest version.
+//. Create a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret] to provide
+//security-related files to the deployer. Providing the secret is optional, and
+//the objects will be randomly generated if not supplied.
+//+
+//You can supply the following files when creating a new secret, for example:
+//+
+//----
+//$ oc create secret generic logging-deployer \
+//    --from-file kibana.crt=/path/to/cert \
+//    --from-file kibana.key=/path/to/key
+//----
+//
+//[cols="3,7",options="header"]
+//|===
+//|File Name
+//|Description
+//
+//|*_openshift_logging_kibana_cert_*
+//|A browser-facing certificate for the Kibana server.
+//
+//|*_openshift_logging_kibana_key_*
+//|A key to be used with the Kibana certificate.
+//
+//|*_openshift_logging_kibana_ops_cert_*
+//|A browser-facing certificate for the Ops Kibana server.
+//
+//|*_openshift_logging_kibana_ops_key_*
+//|A key to be used with the Ops Kibana certificate.
+//
+//|*_openshift_logging_kibana_ops_ca_*
+//|The absolute path on the control node to the CA file to use
+//for the public facing ops kibana certs.
+//
+//|*_server-tls.json_*
+//|JSON TLS options to override the Kibana server defaults. Refer to
+//https://nodejs.org/api/tls.html#tls_tls_connect_options_callback[Node.JS] docs
+//for available options.
+//
+//|*_ca.crt_*
+//|A certificate for a CA that will be used to sign all certificates generated by
+//the deployer.
+//
+//|*_ca.key_*
+//|A matching CA key.
+//|===
 
-====
+[[deploying-the-efk-stack]]
+== Deploying the EFK Stack
 
-[cols="3,7",options="header"]
-|===
-|Parameter Name
-|Description
+The EFK stack is deployed using an Ansible playbook to
+the the EFK components. Run the playbook from the default Openshift Ansible location
+using the default
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[inventory] file.
 
 ifdef::openshift-origin[]
-|*_IMAGE_PREFIX_*
-|The prefix for logging component images. For example, setting the prefix to
-*openshift/origin-* creates *openshift/origin-logging-deployer:v1.2*.
-
-|*_IMAGE_VERSION_*
-|The version for logging component images. For example, setting the version to
-*v1.2* creates *openshift/origin-logging-deployer:v1.2*.
+----
+$ ansible-playbook playbooks/common/openshift-cluster/openshift_logging.yml
+----
 endif::openshift-origin[]
-ifdef::openshift-enterprise[]
-|*_IMAGE_PREFIX_*
-|The prefix for logging component images. For example, setting the prefix to
-*registry.access.redhat.com/openshift3/* creates *registry.access.redhat.com/openshift3/logging-deployer:latest*.
 
-|*_IMAGE_VERSION_*
-|The version for logging component images. For example, setting the version to
-*v3.3* creates *registry.access.redhat.com/openshift3/logging-deployer:v3.3*.
+ifdef::openshift-enterprise[]
+----
+$ /usr/share/ansible/openshift-ansible/playbooks/common/openshift-cluster/openshift_logging.yml
```

Comment:
Oh this still needs the `ansible-playbook` command in front the file name, like in the Origin ifdef. Could also show an optional `-i` in case their inventory file is not at /etc/ansible/hosts:

```
$ ansible-playbook [-i </path/to/inventory>] \
    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-cluster/openshift_logging.yml
```